document/CoreFormat.mdk

```diff
@@ -994,9 +1071,27 @@ This allows tools to include information about the result that is not explicitly
 
 ### General { #location-General }
 
-A `location` object specifies the location where a static analysis tool produced a result.
+A `location` object specifies the location where an analysis tool produced a result.
 A `location` object should specify the physical location (&sect;[#physicalLocation]) of the result,
-and it may also specify the logical location (&sect;[#location-logicalLocation]) of the result.
+and it may also specify the logical location (&sect;[#location-fullyQualifiedLogicalName]) of the result.
+
+A logical location specifies a programmatic construct, for example, a class name or a function name,
+without specifying the programming artifact within which that construct occurs.
+
+~ IsoNote
+There are two reasons to include logical locations in the SARIF format in addition to physical locations:
+
+1. Logical location information is an important contributor to fingerprinting scenarios,
+    because it is more resilient to changes in source code than are line locations.
+    See Annex A for more information about fingerprinting.
+    The `fullyQualifiedLogicalName` property (&sect;[#location-fullyQualifiedLogicalName])
+    is particularly convenient for fingerprinting.
+    
+2. In the absence of symbol information, binary analysis tools might not have source code locations available,
+    so information about line and column numbers might not be present in the log file.
+    In this case, code editors can use logical location to navigate from a result to the correct
```

Comment:
not just code editors but human beings. absent source file details, you need to be able to find whatever an analysis result points to.